base_external_dbsource/README.rst

```diff
@@ -67,6 +71,7 @@ Contributors
 * Maxime Chambreuil <maxime.chambreuil@savoirfairelinux.com>
 * Gervais Naoussi <gervaisnaoussi@gmail.com>
 * Michell Stuttgart <michellstut@gmail.com>
```

Comment:
@lasley please, remove me of contributors. I only provide firebird support. I din't work in this module core code.